translations/de.ts

```diff
@@ -598,69 +579,42 @@ Soll der Proxyserver ignoriert werden und eine Direktverbindung hergestellt werd
 <context>
     <name>Core</name>
     <message>
-        <location filename="../src/core/core.cpp" line="260"/>
+        <location filename="../src/core/core.cpp" line="259"/>
         <source>Toxing on qTox</source>
-        <translation>Tox ist Toll!</translation>
+        <translation>Tox ist toll</translation>
```

Comment:
May I suggest `Toxt mit qTox` as translation. Since `to tox` is not actually an English word (yet?) we may use a Kustwort in German as well. Also `ist toll` sounds kinds of dumb (you wouldn't use `Tox is great/neat` for the source string either).